src/app/page/UserProfile/UserProfile.js

```diff
@@ -1,32 +1,182 @@
 import React from "react";
+import { useSelector } from "react-redux";
 import { withRouter } from "react-router-dom";
 import PropTypes from "prop-types";
 import { Page, Card } from "../../layout";
-import { Chip } from "../../component";
-import { StyledImage, StyledHr } from "./UserProfile.style";
-import { Typography } from "@material-ui/core";
+import { Chip, Button, H5 } from "../../component";
```

Comment:
We definitely need to refactor this file into multiple files later.

Reply:
yeah, I am still working on that, it is a quite a bit left